Source/NSubstitute/SubstituteExtensions.cs

```diff
@@ -222,7 +294,25 @@ internal static ConfiguredCall Returns<T>(MatchArgs matchArgs, Func<CallInfo, T>
         public static IEnumerable<ICall> ReceivedCalls<T>(this T substitute) where T : class
         {
             return GetRouterForSubstitute(substitute).ReceivedCalls();
-        }       
+        }
+
+#if NET4 || NET45
```

Comment:
Might be a separate class would be better for these private methods, what do you think?